expr.go

```diff
@@ -2118,6 +2125,19 @@ func maxValue(f64s []float64, absent []bool) float64 {
 	return m
 }
 
+func minValue(f64s []float64, absent []bool) float64 {
```

Comment:
As `maxValue` is just above, it appears the code is wrong.  Perhaps add a test to catch this sort of bug too.